src/main/java/com/conveyal/datatools/common/status/MonitorableJob.java

```diff
@@ -261,41 +253,68 @@ public void addNextJob(MonitorableJob ...jobs) {
         // Name of file/item once completed
         public String completedName;
 
+        /**
+         * Update status message and percent complete. This method should be used while job is still in progress.
+         */
         public void update (String message, double percentComplete) {
+            LOG.info("Job updated `{}`: `{}`\n{}", name, message, getCallingMethodTrace());
             this.message = message;
             this.percentComplete = percentComplete;
         }
 
-        public void update (boolean isError, String message, double percentComplete) {
-            this.error = isError;
-            this.message = message;
-            this.percentComplete = percentComplete;
+        /**
+         * Gets stack trace from method calling {@link #update(String, double)} or {@link #fail(String)} for logging
+         * purposes.
+         */
+        private String getCallingMethodTrace() {
+            StackTraceElement[] stackTrace = Thread.currentThread().getStackTrace();
+            // Get trace from method calling update or fail. To trace this back:
+            // 0. this thread
+            // 1. this method
+            // 2. Status#update or Status#fail
+            // 3. line where update/fail is called in server job
+            return stackTrace.length >= 3 ? stackTrace[3].toString() : "WARNING: Stack trace not found.";
         }
 
-        public void update (boolean isError, String message, double percentComplete, boolean isComplete) {
+        /**
+         * Shorthand method to update status object on successful job completion.
+         */
+        public void finish(String message) {
```

Comment:
Might be better to rename to something more explicit. Perhaps `completeSuccessfully`.

Reply:
Yea, I was waffling on the name `finish`. Thanks for the recommendation.